installation/proc-install-rhdh-operator.adoc

```diff
@@ -0,0 +1,49 @@
+[id='proc-install-rhdh-operator_{context}']
+= Installing {product} using the Operator
+
+[IMPORTANT]
+====
+The {product} Operator is a Technology Preview feature only.
+
+Technology Preview features are not supported with Red Hat production service level agreements (SLAs), might not be functionally complete, and Red Hat does not recommend using them for production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+For more information on Red Hat Technology Preview features, see https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Scope].
+====
+
+This section describes the installation procedure for installing {product} on your OpenShift Container Platform using the Operator.
+
+//You can use the {product} Operator in Red Hat OpenShift (OpenShift) to install {product}.
+
+.Prerequisites
+
+* You are logged in as an administrator on the OpenShift web console.
+* You have configured the appropriate roles and permissions within your project to create an application. See the link:https://docs.openshift.com/container-platform/4.14/applications/index.html[Red Hat OpenShift documentation on Building applications] for more details.
+* If installing a CI build from quay.io/rhdh, make sure that:
+** you have been granted permission to pull images from the `rhdh` organization on quay.io, and 
+** you have added your Quay token to the cluster global pull secret (link:https://docs.openshift.com/container-platform/4.14/openshift_images/managing_images/using-image-pull-secrets.html#images-update-global-pull-secret_using-image-pull-secrets[link]):
+{code}
+oc get secret/pull-secret -n openshift-config --template='{{index .data ".dockerconfigjson" | base64decode}}' > /tmp/my-global-pull-secret.yaml
+
+oc registry login --registry="quay.io" --auth-basic="<user>:<token>" --to=/tmp/my-global-pull-secret.yaml
+
+oc set data secret/pull-secret -n openshift-config --from-file=.dockerconfigjson=/tmp/my-global-pull-secret.yaml
+
+rm -f /tmp/my-global-pull-secret.yaml
+{code}
+** you have added a link:https://gist.github.com/nickboldt/d54ba9d7264a5480d14eea6c3bd2ecdf[RHDH Operator Catalog Source]:
```

Comment:
This is OK for now but we should move this script into a public place like either THIS repo or the future home of https://github.com/janus-idp/backstage-showcase/ so it's available in the docs. 

Should be fixed before GA... if we can move https://github.com/janus-idp/backstage-showcase/ before 1.1, then use that; if not, then embed a copy of the script in this repo.

Reply:
> or the future home of [janus-idp/backstage-showcase](https://github.com/janus-idp/backstage-showcase/)

You mean "the future home of [janus-idp/operator](https://github.com/janus-idp/operator)", right?

Reply:
yes, installer script makes more sense in the operator repo than the hub repo, as we can also install hub via helm.  Sorry, wrong link paste.